### PR TITLE
Add links to raw YAML specs on API docs

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: api/fern/docs/pages/overview.mdx [hunk 1]
```diff
@@ -62,3 +62,10 @@ If the API returns a `200` or `204` code, your API request was a success. If the
 - `500`: **Internal sever error** - The request could not be completed because of an error caused by Astro.
 
 All error responses include a `requestId` that you can share with Astronomer support if you want to learn more about the error. 
+
+## Download OpenAPI specifications
+
+To read Astro API documentation from another platform, such as Postman or Swagger, download the YAML configurations on the following pages and import them into the tool of your choice.
+
+- [Astro API platform specification](https://api.astronomer.io/spec/platform/v1beta1)
```
Comment (odaneau-astro) [inline api/fern/docs/pages/overview.mdx:70]:
```suggestion
- [Astro API Platform specification](https://api.astronomer.io/spec/platform/v1beta1)
```

File: api/fern/docs/pages/overview.mdx [hunk 2]
```diff
@@ -62,3 +62,10 @@ If the API returns a `200` or `204` code, your API request was a success. If the
 - `500`: **Internal sever error** - The request could not be completed because of an error caused by Astro.
 
 All error responses include a `requestId` that you can share with Astronomer support if you want to learn more about the error. 
+
+## Download OpenAPI specifications
+
+To read Astro API documentation from another platform, such as Postman or Swagger, download the YAML configurations on the following pages and import them into the tool of your choice.
```
Comment (odaneau-astro) [inline api/fern/docs/pages/overview.mdx:68]:
```suggestion
To export Astro API's OpenAPI specifications for use on another platform, such as Postman or Swagger, download the YAML configurations from the following pages and import them into the tool of your choice.
```

**Review by odaneau-astro — COMMENTED** [review]
Looking good. Just added a couple suggestions.

**Review by lzdanski — APPROVED** [review]
After addressing Olivier's comments, LGTM

**Pull request merged**